src/main/java/com/twilio/oai/api/ApiResourceBuilder.java

```diff
@@ -134,51 +140,48 @@ protected Map<String, Object> mapOperation(CodegenOperation operation) {
     }
 
     private void applyListOperation(CodegenOperation operation, Map<String, Object> operationMap, HttpMethod httpMethod) {
-
         if(!operation.vendorExtensions.containsKey("x-ignore")) {
             if (httpMethod == HttpMethod.POST) {
                 operationMap.put("x-name", API_OPERATION_CREATE);
-                operationMap.put("x-is-create-operation", true);
             } else if (httpMethod == HttpMethod.GET) {
                 operationMap.put("x-name", API_OPERATION_READ);
```

Comment:
I meant to put the previous comment here, sorry for the confusion

Reply:
I'm not following. This line didn't change and isn't affecting client generation.